news/urls.py

```diff
@@ -7,22 +7,24 @@
                     unsubscribe, user)
 
 
+TOKEN_RE = '(?P<token>[0-9A-Fa-f-]{36})'
+
 urlpatterns = patterns('',  # noqa
     url('^get-involved/$', get_involved),
     url('^fxa-register/$', fxa_register),
     url('^subscribe/$', subscribe),
     url('^subscribe_sms/$', subscribe_sms),
-    url('^unsubscribe/(.*)/$', unsubscribe),
-    url('^user/(.*)/$', user),
-    url('^confirm/(.*)/$', confirm),
+    url('^unsubscribe/{0}/$'.format(TOKEN_RE), unsubscribe),
+    url('^user/{0}/$'.format(TOKEN_RE), user),
+    url('^confirm/{0}/$'.format(TOKEN_RE), confirm),
```

Comment:
I like this change, I just have a small suggestion to make it slightly more readable would be to add a helper function that expects the unformated string, something like

```python
def token_url(url_format, *args, **kwargs):
    return url(url_format.format(TOKEN_RE), *args, **kwargs)

# ...

urlpatterns = patterns(
    # ...
    token_url('^unsubscribe/{0}/$', unsubscribe),
    token_url('^user/{0}/$', user),
    # ...
```

If you prefer it the way you have it that's fine with me too. :)